src/skills-builder/skills-builder-modal/select-preferences/CareerInterestSelect.jsx

```diff
@@ -16,25 +18,24 @@ const CareerInterestSelect = () => {
   const { searchClient } = algolia;
 
   const handleCareerInterestSelect = (value) => {
-  // By checking for a value to exist, we avoid adding a null value to the careerInterests array
-  // The 'onSelected' function is fired during every 'onChange' event
-  // A null value was being passed to this function whenever the search box received input, resulting in empty cards
-    if (value && careerInterests.length < 3) {
+    if (!careerInterests.includes(value) || careerInterests.length < 3) {
       dispatch(addCareerInterest(value));
     }
   };
 
   return (
     <Stack gap={2}>
-      <h4>
-        {formatMessage(messages.careerInterestPrompt)}
-      </h4>
-      <InstantSearch searchClient={searchClient} indexName={getConfig().ALGOLIA_JOBS_INDEX_NAME}>
-        <JobTitleInstantSearch
-          onSelected={handleCareerInterestSelect}
-          placeholder={formatMessage(messages.careerInterestInputPlaceholder)}
-        />
-      </InstantSearch>
+      <Form.Label>
+        <h4 className="mb-3">
+          {formatMessage(messages.careerInterestPrompt)}
+        </h4>
+        <InstantSearch searchClient={searchClient} indexName={getConfig().ALGOLIA_JOBS_INDEX_NAME}>
+          <JobTitleInstantSearch
+            onSelected={handleCareerInterestSelect}
+            placeholder={formatMessage(messages.careerInterestInputPlaceholderText)}
+          />
+        </InstantSearch>
+      </Form.Label>
```

Comment:
This `Form.Autosuggest` didn't have a label associated with it's `input` element. In order to give it one, we can wrap it in `Form.Label` and [associate a label implicitly](https://www.w3.org/TR/html4/interact/forms.html#h-17.9.1:~:text=To%20associate%20a,LABEL%3E%0A%3C/P%3E%0A%3C/FORM%3E).